xmodule/modulestore/tests/mongo_connection.py

```diff
@@ -9,5 +9,4 @@
 import os
 
 MONGO_PORT_NUM = int(os.environ.get('EDXAPP_TEST_MONGO_PORT', '27017'))
-MONGO_HOST = os.environ.get('EDXAPP_TEST_MONGO_HOST',
-                            'edx.devstack.mongo' if 'BOK_CHOY_HOSTNAME' in os.environ else 'localhost')
+MONGO_HOST = os.environ.get('EDXAPP_TEST_MONGO_HOST', 'localhost')
```

Comment:
I think the devstack reference is just because bokchoy _used_ devstack (well, I'm guessing it did). As far as I can tell, this would never be set during normal devstack usage. So this change is probably fine?